source/adapters/cuda/image.cpp

```diff
@@ -430,6 +430,11 @@ UR_APIEXPORT ur_result_t UR_APICALL urBindlessImagesImageFreeExp(
   ScopedContext Active(hDevice);
   try {
     UR_CHECK_ERROR(cuArrayDestroy((CUarray)hImageMem));
+    if (auto it = hDevice->ChildCuarrayFromMipmapMap.find((CUarray)hImageMem);
```

Comment:
Not a huge fan of c-style casts and in other parts of the codebase we try to actively avoid them, but seems like in this file it's a done thing, so am OK with it.

Reply:
I agree, we'll need to get rid of the C-style casts separately.